editor.go

```diff
@@ -17,53 +17,77 @@ func NewEditor() *Editor {
 
 //go:generate gentut -t Editor -p owasp -in editor.go -w
 type Editor struct {
-	entries []Entry
+	Entries []Entry
```

Comment:
What would be the argument for this exposure? we talked about it making it easier for others to make arbitrary changes to the entries but at the cost of each method not being able to guarantee concurrent access. Please add your reasoning why you think it should be public.